### PR TITLE
chore: add contribution guidelines

### LINKED ISSUE #128: No contribution guidelines.
**Describe the chore**
There should be contribution guidelines for developers in the docs.

**Would you like to work on the issue?**
Yes.

### PR DESCRIPTION
**Pull request by naman653**
Fixes #128

### CODE REVIEW

File: docs/commitStyle.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+## Commit Style Guidelines for Susi Android
```
Comment (yashk2000) [inline docs/commitStyle.md]: For `Susi Android`? Change it to Badge-Magic Android.

Comment (adityastic) [reply docs/commitStyle.md]: XD

Comment (naman653) [reply docs/commitStyle.md]: 😅

File: docs/ContributionHelp.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+## How to Contribute
```
Comment (iamareebjamal) [inline docs/ContributionHelp.md]: Rename to CONTRIBUTION.md and move to root

**Pull request merged**